### PR TITLE
Exams

### PR DESCRIPTION
**Pull request by erotourtes**
What do You think I can improve?

### CODE REVIEW

File: Tasks/quotes.js [hunk 1]
```diff
@@ -1,30 +1,26 @@
+'use strict';
 // Change double quotation to open or close quotation
 
-const EMPTY = '';
+const quotes = (str) => {
```
Comment (pavukach) [inline Tasks/quotes.js:4]: Will be better to remove parentheses around `str`

File: Tasks/quotes.js [hunk 2]
```diff
@@ -1,30 +1,26 @@
...
+  const res = [];
```
Comment (pavukach) [inline Tasks/quotes.js]: You can create an array with `new Array()` or `new Array(str.length)`

File: Tasks/words.js [hunk 3]
```diff
@@ -1,41 +1,13 @@
...
-          flag = true;
-        } else {
-          flag = true;
-        }
-      }
-    }
-  }
-  return number_of_words_in_s;
+const numOfWords = (str) => {
+  const isNotBlank = (char) => char !== '';
```
Comment (pavukach) [inline Tasks/words.js:5]: here too

**Review by pavukach — COMMENTED** [review]
Here some possible changes

File: Tasks/quotes.js (hunk 1 above)
Comment (erotourtes) [reply Tasks/quotes.js:4]: Our `.eslintrc.json` file uses the option 
```json
"arrow-parens": [
      "error",
      "always"
    ],
```
So we need to keep it in one style

File: Tasks/quotes.js (hunk 2 above)
Comment (erotourtes) [reply Tasks/quotes.js]: Yeah, I think it would be better

**Comment by kol-oss** [comment]
Nothing to add

**Pull request merged**